app/Http/Controllers/Admin/ReservationController.php

```diff
@@ -123,4 +123,23 @@ public function update(UpdateReservationRequest $request, $id)
             return redirect()->route('reservation.edit');
         }
     }
+
+    /**
+     * Delete a booking rooms.
+     *
+     * @param int $id of reservation
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        $reservationDelete = Reservation::findOrFail($id)->delete();
+        if ($reservationDelete) {
+            flash(__('Delete Booking Room Success!'))->success();
+            return redirect()->route('reservation.index');
+        } else {
+            flash(__('Delete Booking Room Fail!'))->error();
+            return redirect()->route('reservation.index');
+        }
```

Comment:
You should add `return redirect()->route('reservation.index');` at here
and remove same code in if...else